modules/core/src/main/java/org/locationtech/jts/geom/impl/CoordinateArraySequence.java

```diff
@@ -18,13 +18,12 @@
 /**
  * A {@link CoordinateSequence} backed by an array of {@link Coordinate}s.
  * This is the implementation that {@link Geometry}s use by default.
- * Coordinates returned by #toArray and #getCoordinate are live --
+ * Coordinates returned by <code>toCoordinateArray</code> and <code>getCoordinate</code> are live --
```

Comment:
I tended to use `@code` for in-line stuff:
```
  * Coordinates returned by {@code toCoordinateArray()}  and {@code getCoordinate()} are live --
```

But in this case you can link to the actual methods:

```
  * Coordinates returned by {@link #toCoordinateArray()}  and {@link #getCoordinate()} are live --

```

Reference:

* https://reflectoring.io/howto-format-code-snippets-in-javadoc/

Reply:
Yup, can do that.